openedx/features/badging/handlers.py

```diff
@@ -12,7 +15,7 @@
 
 
 @receiver(post_save, sender=Badge)
-def sync_badge_info_with_nodebb(sender, instance, update_fields, **kwargs):
+def sync_badge_info_with_nodebb(sender, instance, update_fields, **kwargs):  # pylint: disable=unused-argument
```

Comment:
Instead of disabling this pylint warning in 4 different places within the file, we can do it once at the start, before the import statements, since there are total of 4 functions in this file, and this warning is to be disabled for all 4 functions. WDYT?

Reply:
I think it is fine to disable for these places only instead of disabling for whole files.